modules/mono/glue/cs_files/Color.cs

```diff
@@ -342,7 +342,7 @@ private static int _parse_col(string str, int ofs)
             for (var i = 0; i < 2; i++)
             {
                 int c = str[i + ofs];
-                var v = 0;
+                int v;
```

Comment:
I am **not** a fan of changing things to `var`, at all. It makes the type ambiguous, which is confusing and leads to problems. You've already seen one of the issues with using `var` in this commit, as when you removed default assignments, you had to change the types back to explicit. It can also cause other problems like confusion. I vote for *not* changing everything to `var`.

However, I approve of most of your other changes.

Reply:
The main use case of `var` is when it makes code more readable due to avoiding repetition. `var f = new MyVeryLongClassName();` is better than `MyVeryLongClassName f = new MyVeryLongClassName();`. But, replacing `int` with `var` makes the code less readable.

Reply:
Personally, I do not see it as an issue, but I understand your position and I'm willing to change it if majority agree with your opinion.

There's an ongoing discussion on this matter in #godotengine/godot-docs#1296, so I'd suggest to continue the discussion there.

Reply:
https://github.com/godotengine/godot-docs/issues/1296#issuecomment-379522729

Reply:
Thanks :)